common/cluster/metadata.go

```diff
@@ -91,8 +91,12 @@ type (
 		MasterClusterName string `yaml:"masterClusterName"`
```

Comment:
Comment is not accurate. Only master can register namespace, all cluster can update namespace. We probably should consider deprecate master cluster concept.

Reply:
Not plan to do this in this PR. But I agree we should start a thread to deprecate this.